java/code/src/com/redhat/rhn/frontend/taglibs/DateTimePickerTag.java

```diff
@@ -207,7 +207,7 @@ private void writePickerHtml(Writer out) throws IOException {
         out.append(createHiddenInput("minute", String.valueOf(data.getMinute())).render());
         if (data.isLatin()) {
             out.append(createHiddenInput("am_pm",
-                    String.valueOf((data.getHourOfDay() > 12) ? 1 : 0)).render());
+                    String.valueOf((data.getHourOfDay() >= 12) ? 1 : 0)).render());
```

Comment:
Oof, this is wicked. I took this logic straight over to the legacy picker frontend and didn't notice the error either. Good job finding and fixing this. 👍